lzero/model/unizero_world_models/lpips.py

```diff
@@ -21,16 +21,16 @@ def __init__(self, use_dropout: bool = True):
         self.scaling_layer = ScalingLayer()
         self.chns = [64, 128, 256, 512, 512]  # vg16 features
         # Comment out the following line if you don't need perceptual loss
-        self.net = vgg16(pretrained=True, requires_grad=False)
+        # self.net = vgg16(pretrained=True, requires_grad=False)
```

Comment:
why comment this part

Reply:
to be more efficient (omit the model load process) when we don't use decode regularization.